splitio/client/input_validator.py

```diff
@@ -92,14 +93,14 @@ def _check_string_matches(value, operation, pattern):
     :return: The result of validation
     :rtype: True|False
     """
-    if not re.match(pattern, value):
+    if re.search(pattern, value) is None or re.search(pattern, value).group() != value:
         _LOGGER.error(
             '%s: you passed %s, event_type must ' +
             'adhere to the regular expression %s. ' +
-            'This means an event name must be alphanumeric, cannot be more ' +
+            'This means %s must be alphanumeric, cannot be more ' +
             'than 80 characters long, and can only include a dash, underscore, ' +
```

Comment:
what about 80 characters?

Reply:
for flagSets it's 50

Reply:
Will add the character cunt as a parameter